rpcs3/Emu/Memory/vm_locking.h

```diff
@@ -52,6 +52,14 @@ namespace vm
 		// Old-style conditional constexpr
 		const u32 size = Size ? Size : _size;
 
+		// Assume address is aligned to size if less/equal than 16-bytes
+		if (Size - 1 <= 15u ? !vm::check_addr(begin) : !vm::check_addr(begin, vm::page_readable, size))
```

Comment:
Why max 16? I think it can be 4096. Plus you don't have to optimize constexpr checks.